python/trezorctl

```diff
@@ -113,6 +113,14 @@ CHOICE_OUTPUT_SCRIPT_TYPE = ChoiceType(
     }
 )
 
+CHOICE_RESET_DEVICE_TYPE = ChoiceType(
+    {
+        "single": proto.ResetDeviceType.Bip39,
+        "shamir": proto.ResetDeviceType.Slip39_Single_Group,
+        "shamir-advanced": proto.ResetDeviceType.Slip39_Multiple_Groups,
```

Comment:
let's call the last one just `advanced`
maybe move to legacy - default - advanced ?

Reply:
The Single / Shamir backup terms are somewhat "official". In other words, they were agreed on with marketing etc. and will be used in all the marketing materials. So it makes sense to have it consistent.

I agree we can have "advanced" only for now and rethink when the multiple groups support is actually implemented.

Reply:
I would remove the multiple groups option from trezorctl at this moment. Currently it serves no purpose. We can decide on the naming when the time comes.

Reply:
i'd say if the value exists in the proto definition, let's have it in trezorctl as well

Reply:
Ok "advanced" for now and let's see later.